validationAndData.js

```diff
@@ -0,0 +1,17 @@
+const emailValidation = () => {
+  const eemail = document.getElementById('email').value;
+  const regExp = /^([a-z\d-]+)@([a-z\d-]+)\.([a-z]{2,8})(\.[a-z]{2,8})?$/;
+  document.forms[0].onsubmit = (e) => {
+    if (regExp.test(eemail)) {
+      document.getElementById('error-message').innerHTML = '';
+    } else {
+      alert(regExp.test(eemail));
```

Comment:
- [ ] Great job adding the validation form 💯, but when you enter an email with an uppercase letter, an alert message appears. Kindly, remove it because it is not present in the original design. 🏞️

![image](https://user-images.githubusercontent.com/91301423/204097014-900edbee-284e-440b-a69c-322162b3715d.png)